packages/office-addin-manifest/src/addInTypes.ts

```diff
@@ -48,7 +50,7 @@ export function parseAddInType(value: string): AddInType {
   const addInType = toAddInType(value);
 
   if (!addInType) {
-    throw new Error(`${value} is not a valid Office add-in type.`);
+    throw new ExpectedError(`${value} is not a valid Office add-in type.`);
```

Comment:
Is this really expected?  This method is just converting a string to enum and has no idea on the context of where that string came from.

Reply:
Yes, it is parsing what the user typed for the argument.

Reply:
That may be in intended use, but the method could be used in unexpected ways.

Reply:
If this is called with a string that doesn't return an add-in type, is this a problem with our code? No. It reflects in the telemetry that an error is expected in this situation given the inputs.

Reply:
It may not be a problem with the code in this method, but it could be a problem with some other code of ours that starts to call this method.  There isn't enough context in this method to indicate expected or not . . . that would have to be determined by the caller.

Reply:
I guess if another caller would use that function in which it could raise an unexpected error, this caller could just catch this error and then parse it to be a normal error.
I feel most of the time this function should return an expected error given that the main error is passing an invalid string.

Reply:
That requires the caller to know that the method is returning an expected error and that it needs to be changed to be something not expected.  Seems like a caller that wants it to be expected should be the one to catch it and make is expected.  Again . . . the method has no context to determine expected or not.

Reply:
The caller will get an exception that there is an error. The ExpectedError is so our telemetry can distinguish that the error is expected (given the input).

Reply:
I fell like putting to be just error is the safe approach, given that we generally won't be looking that much into expected errors in the telemetry.
Because of all of this discussion, I fell like just `error` would the right way.

Reply:
Let's discuss offline.